.github/workflows/staging-smoke-tests.yaml

```diff
@@ -61,6 +61,7 @@ jobs:
           cd ${GITHUB_WORKSPACE}/${workingDir}
           echo "Building Data Repo client library"
           export TEST_RUNNER_SERVER_SPECIFICATION_FILE="staging.json"
+          export TEST_RUNNER_BILLING_PROFILE_NAME="stest1"
```

Comment:
If this is specific to environment, I do think it would be better to define it in the TEST_RUNNER_SERVER_SPECIFICATION_FILE, like the "staging.json" file listed above. We already use this for setting the billing account Id, so I don't see why we'd deviate for the billing profile.

Reply:
oh, can you specify arbitrary things in there?

Reply:
Is it strange that we would have billingAccount in the test definition and billingProfileName in server?  Should that be in the test config?

Reply:
Eh, looks like it'll be easier to do from within the server

Reply:
And that seems to be how it's intended from the code

Reply:
Oh yeah - you're totally right. We define the billing account right alongside the server. https://github.com/DataBiosphere/jade-data-repo/blob/develop/datarepo-clienttests/src/main/java/runner/TestRunner.java#L185

I'm less convinced of my stance now 😄  - I think the environment variable is fine, I don't think we want it in the test config.

For the server specification, we basically have it defined in a bunch of places (including the test config), with an override order. https://github.com/DataBiosphere/jade-data-repo/tree/develop/datarepo-clienttests#override-the-server-from-the-command-line
I don't like this b/c it's easy to get confused where something is defined. So, just an environment variable sounds good.